### PR TITLE
[ios] Fixed crash while deleting bookmark.

### PR DESCRIPTION
**Pull request by VladiMihaylenko**
https://jira.mail.ru/browse/MAPSME-3866

### CODE REVIEW

File: iphone/Maps/Bookmarks/BookmarksRootVC.mm [hunk 1]
```diff
@@ -9,6 +9,10 @@
 
 #define TEXTFIELD_TAG 999
 
+extern NSString * const kBookmarkCategoryDeletedNotification =
```
Comment (syershov) [inline iphone/Maps/Bookmarks/BookmarksRootVC.mm]: Extern keyword is not need here.

File: iphone/Maps/Bookmarks/BookmarksVC.mm [hunk 2]
```diff
@@ -27,6 +27,7 @@
 #define EMPTY_SECTION -666
 
 extern NSString * const kBookmarksChangedNotification = @"BookmarksChangedNotification";
+extern NSString * const kBookmarkDeletedNotification = @"BookmarkDeletedNotification";
```
Comment (syershov) [inline iphone/Maps/Bookmarks/BookmarksVC.mm]: Here too.

Comment (VladiMihaylenko) [inline iphone/Maps/Bookmarks/BookmarksVC.mm]: __private_extern__ is enabled in our project

File: iphone/Maps/Bookmarks/BookmarksRootVC.mm (hunk 1 above)
Comment (syershov) [inline iphone/Maps/Bookmarks/BookmarksRootVC.mm]: It's really need in our project, because we set GCC_SYMBOLS_PRIVATE_EXTERN.

**Pull request merged**